### PR TITLE
Adding sync_file_range Syscall

### PR DESCRIPTION
**Pull request by kuzeyardabulut**

### CODE REVIEW

File: sysdeps/nacl/sysdep.h [hunk 1]
```diff
@@ -2035,19 +2035,17 @@ INTERNAL_SYSCALL_sync_0 (int *err)
...
-#if 0
-  __off64_t offset = ((__off64_t)offset_high) << 32 | offset_low;
-  __off64_t nbytes = ((__off64_t)nbytes_high) << 32 | nbytes_low;
-#endif
-  log_unimplemented("sync_file_range unimplemented");
-  *err = (38 /* ENOSYS */);
-  return 0;
+
+{ 
+ int rv = __nacl_irt_sync_file_range (fd, offset, nbytes, flags);
```
Comment (rennergade) [inline sysdeps/nacl/sysdep.h:2043]: weird spacing here too

**Review by rennergade — CHANGES_REQUESTED** [review]
Some nitpicks about whitespace. Content looks good.

**Review by RusherRG — APPROVED** [review]
Apart from the formatting changes suggested by @rennergade, everything else looks good.

**Review by justyoyo123 — APPROVED** [review]
This looks good to me.

**Pull request merged**